prisma/schema.prisma

```diff
@@ -11,13 +11,15 @@ datasource db {
 }
 
 model User {
-  id                  Int                   @id @default(autoincrement())
-  email               String                @unique
-  password            String
-  role                Role                  @default(USER)
-  recipes             Recipe[]
-  webhooks            Webhook[]
-  personalAccessToken PersonalAccessToken[]
+  id                     Int                   @id @default(autoincrement())
+  email                  String                @unique
+  password               String
+  role                   Role                  @default(USER)
+  activated              Boolean               @default(false)
+  accountActivationToken String?
```

Comment:
I would move it to a separate table.

Before user activates his/her account, there should be no `User` record created (no account should be created yet).

We could however assume that only `activated === true` accounts are created / active accounts, but then when developing other features related to user, e.g. "invite user to organisation" we would have to remember not to do a simple `select * from User` but add a `where` clause where we would filter out not activated accounts. What I'm trying to say - moving this to a separate table - although not necessary - might save us some trouble later.

After activation we can either remove the record from the new table or leave the record but keep only the necessary data (so e.g. remove the password).

Also, I would rename `activated` to `isActivated`